.travis.yml

```diff
@@ -0,0 +1,314 @@
+            - gcc-4.9
+          sources:
+            - ubuntu-toolchain-r-test
+
+    - env: CLANG_VERSION=3.7 CMAKE_BUILD_TYPE=RELWITHDEBINFO BITS=32
+      os: linux
+      addons:
+        apt:
+          packages:
+            - clang-3.7
+            - g++-multilib
+            - gcc-multilib
+            - libc6-dev-i386
+          sources:
+            - llvm-toolchain-precise
+            - ubuntu-toolchain-r-test
+
+    - env: CLANG_VERSION=3.6 CMAKE_BUILD_TYPE=RELWITHDEBINFO BITS=32
+      os: linux
+      addons:
+        apt:
+          packages:
+            - clang-3.6
+            - g++-multilib
+            - gcc-multilib
+            - libc6-dev-i386
+          sources:
+            - llvm-toolchain-precise-3.6
+            - ubuntu-toolchain-r-test
+
+    - env: CLANG_VERSION=3.5 CMAKE_BUILD_TYPE=RELWITHDEBINFO BITS=32
+      os: linux
+      addons:
+        apt:
+          packages:
+            - clang-3.5
+            - g++-multilib
+            - gcc-multilib
+            - libc6-dev-i386
+          sources:
+            - llvm-toolchain-precise-3.5
+            - ubuntu-toolchain-r-test
+
+    - env: CLANG_VERSION=3.4 CMAKE_BUILD_TYPE=RELWITHDEBINFO BITS=32
+      os: linux
+      addons:
+        apt:
+          packages:
+            - g++-multilib
+            - gcc-multilib
+            - libc6-dev-i386
+          sources:
+            - ubuntu-toolchain-r-test
+
+    - env: GCC_VERSION=5 CMAKE_BUILD_TYPE=RELWITHDEBINFO BITS=32
+      os: linux
+      addons:
+        apt:
+          packages:
+            - g++-5
+            - g++-5-multilib
+            - gcc-5
+            - gcc-5-multilib
+            - linux-libc-dev:i386
+          sources:
+            - ubuntu-toolchain-r-test
+
+    # END GENERATED
+ 
+install:
+  - if [[ "$GCC_VERSION" != "" ]]; then export CC="gcc-$GCC_VERSION" CXX="g++-$GCC_VERSION"; fi
+  - if [[ "$CLANG_VERSION" != "" ]]; then export CC="clang-$CLANG_VERSION" CXX="clang++-$CLANG_VERSION"; fi
+  # clang 3.4 is the default version of clang in Travis CI.
+  - if [[ "$CLANG_VERSION" == "3.4" ]]; then export CC=clang CXX=clang++; fi
+
+  - git clone -b OpenSSL_1_0_2-stable --depth 1 --single-branch https://github.com/openssl/openssl build/openssl
+  - git clone -b master --depth 1 --single-branch https://github.com/briansmith/googletest build/gtest
+
+before_script:
+  # The version specifications leave some ambiguity over the precise version of
+  # the compiler, so clarify which exactly which versions are being used.
+  - $CC --version
+  - $CXX --version
+  - echo CMAKE_BUILD_TYPE is "$CMAKE_BUILD_TYPE"
+
+script:
+  - make openssl-configure CMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE BITS=$BITS
+  - make openssl CMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE BITS=$BITS
+  - make CMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE BITS=$BITS
+  - make check CMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE BITS=$BITS
+  - git clone -b master --depth 1 --single-branch https://github.com/briansmith/googletest build/gtest
```

Comment:
It's just a mistake. I've already corrected it locally. I am using this same build system for https://github.com/briansmith/ring and it's already been corrected there. I'll update the PR soon.